pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/models/consumer/objects_api/uuid/PNUUIDMetadata.java

```diff
@@ -51,16 +53,16 @@ public static PNUUIDMetadata from(com.pubnub.api.models.consumer.objects.uuid.PN
         if (data == null) {
             return null;
         }
-        PNUUIDMetadata newData = new PNUUIDMetadata(data.getId(), data.getName() != null ? data.getName().getValue() : null)
-                .setProfileUrl(data.getProfileUrl() != null ? data.getProfileUrl().getValue() : null)
-                .setEmail(data.getEmail() != null ? data.getEmail().getValue() : null)
-                .setExternalId(data.getExternalId() != null ? data.getExternalId().getValue() : null)
-                .setStatus(data.getStatus() != null ? data.getStatus().getValue() : null)
-                .setType(data.getType() != null ? data.getType().getValue() : null)
-                .setCustom(data.getCustom() != null ? data.getCustom().getValue() : null);
+        PNUUIDMetadata newData = new PNUUIDMetadata(data.getId(), data.getName())
+                .setProfileUrl(data.getProfileUrl())
+                .setEmail(data.getEmail())
+                .setExternalId(data.getExternalId())
+                .setStatus(data.getStatus())
+                .setType(data.getType())
+                .setCustom(data.getCustom() != null ? PatchValue.of(data.getCustom().getValue()) : null);
```

Comment:
Is this correct? Shouldn't be:
`.setCustom(data.getCustom());`
?

Reply:
you're right, I missed this. fortunately it does exactly the same thing, but I'll change it